apps/08jquery-towers-of-hanoi/script.js

```diff
@@ -2,4 +2,34 @@
 
 $(document).ready(function() {
   // Put app logic here
-}
+  var allStacks = 'div[data-stack]';
+  var block = null;
+
+  function checkForWin(){
+    if (($(this).data('stack') > 1) && ($(this).children().length) === 4){
+      $('#announce-game-won').text('You Won!');
+    }
+  }
```

Comment:
@jasonburks Your `checkForWin()` doesn't work. `this` is not defined when you call it.